.github/workflows/verify-build.yaml

```diff
@@ -152,6 +152,29 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
+  template-snapshots-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Run test
+        run: npm run test -w packages/adapters-cli
+        env:
+          DEFI_ADAPTERS_LOG_LEVEL: warn
```

Comment:
Move this to its own job, as it's now in a different package and it was difficult to open the logs on the previous job due to the size of this test.